lib/templates.rake

```diff
@@ -103,6 +107,23 @@ def update_snippet
   { :diff => diff, :status => status, :result => result }
 end
 
+def get_default_branch
+  # Always use develop on Foreman-nightly
+  return "-b develop" if Foreman::Version.new.tag == "develop"
+
+  url = URI.parse("https://api.github.com/repos/theforeman/community-templates/branches")
```

Comment:
This may be an obscure use case, but never underestimate what create solutions users can come up with. An organization could have a production setup and a separate lab setup to test any changes. They have provisioning templates stored in their own git repository to sync them between the two setups. Now the lab has a branch lab with the latest changes while production uses production.

Now that I think about it, it's a bigger problem. If they have a custom git repo, the default branch may exist in our github but not in theirs. I don't think we can assume a repo layout for all our users.